docs/manual/other-topics/upgrade-to-v7.md

```diff
@@ -16,6 +16,18 @@ As a result, the manual typings that were formerly best-effort guesses on top of
 have been removed and all typings are now directly retrieved from the actual TypeScript code.
 You'll likely find many tiny differences which however should be easy to fix.
 
+## Attributes cannot start and end with `$`, or include `.`
```

Comment:
Maybe good to add an example of how people can change the names of the attributes they are using?

Reply:
Done! :)